cms/bundles/viewsets/bundle.py

```diff
@@ -245,7 +245,7 @@ def get_pages_for_manager(self) -> "SafeString":
         )
```

Comment:
adding here as `bundles/viewsets/bundle_page_chooser.py` was not touched. Testing this locally, the chooser is asking for a filter on page type. Something like:
[page_type_filter.patch](https://github.com/user-attachments/files/20186670/page_type_filter.patch)

Reply:
Thanks, added in [05bd5ac](https://github.com/ONSdigital/dis-wagtail/pull/183/commits/05bd5ac3a07aa56c0fe678cd3242d8620098a0b8)

Reply:
This addition needs a test or two ;)